apps/andi/lib/andi_web/live/submit_live_view/review_submission.ex

```diff
@@ -0,0 +1,77 @@
+defmodule AndiWeb.SubmitLiveView.ReviewSubmission do
+  @moduledoc """
+  LiveComponent reviewing submission
+  """
+  use Phoenix.LiveView
+  require Logger
+
+  def mount(_, %{"dataset" => dataset}, socket) do
+    AndiWeb.Endpoint.subscribe("toggle-visibility")
+
+    {:ok,
+     assign(socket,
+       changeset: %{valid?: true, changes: %{}},
```

Comment:
I think you can get rid of this I don't see it being used anywhere

Reply:
yah, it was messing up something in the form_section, but since it's no longer needed it can be removed